### PR TITLE
Some cleanup for mesh expansion.

### PR DESCRIPTION
**Pull request by gyliu513**
1) Added an example ILB
2) Port 27017 is not needed.

### CODE REVIEW

File: _docs/setup/kubernetes/mesh-expansion.md [hunk 1]
```diff
@@ -38,13 +38,14 @@ You should customize it based on your provisioning tools and DNS requirements.
 ### Preparing the Kubernetes cluster for expansion
 
 *   Setup Internal Load Balancers (ILBs) for Kube DNS, Pilot, Mixer and Citadel. This step is specific to
-each cloud provider, so you may need to edit annotations.
+each cloud provider, so you may need to edit annotations. You can use an ILB based on keeplived at
```
Comment (geeknoid) [inline _docs/setup/kubernetes/mesh-expansion.md]: keeplived -> keepalived

Comment (gyliu513) [reply _docs/setup/kubernetes/mesh-expansion.md]: Done

**Comment by gyliu513** [comment]
ping @geeknoid , can this get merged?

**Comment by geeknoid** [comment]
Presubmit test isn't passing. You need to add keepalived to the dictionary (.settings file at the root)

**Comment by gyliu513** [comment]
@geeknoid did not find the `.settings` file, can you please show me a link for this?

**Comment by linsun** [comment]
I'm interested in understanding how to add a word to bypass spelling check too.  cc @geeknoid 

@gyliu513 a little concerned with the keepalived off your own repo in master.  I think it'd be helpful to explain why people wants to use the test approach based on keepalived.  Is it when cloud provider doesn't provide service type load balancer for private cloud IPs?

**Comment by gyliu513** [comment]
@linsun currently, only public cloud can provide loadbalancer services, such as AWS etc, but there does not seems any solutions can be used by on-prem clusters, so here I was providing an example keepalived chart, this can help the customer to do some test and demo if they do not have a loadbalancer, hope it is OK.

**Comment by linsun** [comment]
@gyliu513 I see, thanks, would be good to mention why users would use it, e.g. in the case where the cloud provider or private cloud (for example ICP) doesn't support k8s service type load balancer service.

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit c549144176ff348b9307adddc7dd20f22f4725b8

https://app.netlify.com/sites/preliminary-istio/deploys/5b0eb0d1dd6a546397ec028f

**Comment by geeknoid** [comment]
Sorry, I didn't see these comments earlier.

The dictionary of exceptions is located at the root of the istio.github.io in .settings. Just add a word in there when needed. Note that often, words like keepalived should be in backticks `. When you put something in backticks, it is automatically excluded and not spell checked.

https://github.com/istio/istio.github.io/blob/master/.spelling

**Comment by gyliu513** [comment]
ping @geeknoid @linsun , all comments are now addressed.

**Pull request merged**